src/shell/oom.c

```diff
@@ -20,7 +20,9 @@
 #endif
 #include <sys/types.h>
 #include <sys/stat.h>
+#ifdef HAVE_SYS_INOTIFY_H
```

Comment:
It seems like this plugin should just be conditionally not compiled on macos, since we know cgroups are never supported there? However, perhaps this way is just easier since you don't have to modify the bulitin plugin list?

Reply:
Well that seems obvious now that you mention it. Let me try that way.